src/python/WMCore/WMSpec/Steps/Executors/StageOut.py

```diff
@@ -136,48 +135,40 @@ def execute(self, emulator = None):
             # for that step; or so I hope
             files = stepReport.getAllFileRefsFromStep(step = step)
             for fileName in files:
-                if not hasattr(fileName, 'lfn') and hasattr(fileName, 'pfn'):
-                    # Then we're truly hosed on this file; ignore it
-                    msg = "Not a file: %s" % fileName
+
+                # make sure the file information is consistent
+                if hasattr(fileName, 'pfn') and ( not hasattr(fileName, 'lfn') or not hasattr(fileName, 'module_label') ):
```

Comment:
I'm not sure I follow this code (and the previous one). Don't we care about the 'lfn' attr only? Well, and maybe the 'module_label' for future jobs (or dbs info?)? Why do we need this check for 'pfn'?